cmd/config_test.go

```diff
@@ -142,3 +143,41 @@ func TestResolveDeprecations(t *testing.T) {
 		t.Fatal("invalid config did not cause correct error")
 	}
 }
+
+func TestConfigValidation(t *testing.T) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	config, err := LoadConfig(path.Join(cwd, "..", "config", "example.yaml"))
```

Comment:
Somewhat conflicted about using an example file as a test input. It does make sure that the example is valid to some degree. On the other hand it is really unexpected to have an example be part of a test case.

I do not have much experience with generated documentation, but I feel like this would be another potential use case for it (besides the documentation of supported AWS resources).

Reply:
I had the same doubts. I added another test config for the tests.